.github/workflows/staging.yml

```diff
@@ -40,11 +40,7 @@ jobs:
 
 
       - name: Checkout public vuln-list-redhat-repo
-        uses: actions/checkout@v3
```

Comment:
actions/checkout has `sparse-checkout` flag.
IIUC you can use this.

example for `contrib` dir from Trivy - https://github.com/aquasecurity/setup-trivy/blob/9ea583eb67910444b1f64abf338bd2e105a0a93d/action.yaml#L66-L67

Reply:
I prefer to use "same" logic if possible.
For other repos we use `actions/checkout`, so i would prefer to use `actions`checkout` here.
And for makefile we can leave new logic to simplify work locally

Reply:
I wrote it this way since I thought the "native" way to clone repos within GitHub CI was to use other GitHub actions. We can certainly change it.

Reply:
Got it. I think @nikpivkin is right and it makes sense to switch to makefile.
e.g., Trivy-db already uses `makefile` in `cron` workflow - https://github.com/aquasecurity/trivy-db/blob/main/.github/workflows/cron.yml

But i think it's better to do it in another PR.